pom.xml

```diff
@@ -176,6 +177,18 @@
             <artifactId>oauth-provider</artifactId>
             <version>${net.oauth.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-core</artifactId>
+            <version>2.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-library</artifactId>
+            <version>2.2</version>
```

Comment:
I _assume_ these are versioned together? If so can we please extract the version out into a property, so we can bump it in one place rather than two when it becomes time to upgrade